scripts/update_feedstocks_submodules.py

```diff
@@ -40,8 +42,16 @@
             feedstocks_repo.submodules[feedstock.package_name].remove()
 
         # Add the new submodule.
-        feedstocks_repo.create_submodule(feedstock.package_name, repo_subdir,
-                                         url=feedstock.clone_url)
+        try:
+            feedstocks_repo.create_submodule(feedstock.package_name, repo_subdir,
+                                             url=feedstock.clone_url)
+        except ValueError:
+            warning.warn(
+                    "Unable to add the submodule {}. "
+                    "This is likely because the repo has no commits, "
+                    "which likely means something went wrong with feedstock generation. "
+                    "Will skip adding this submodule and continue.".format(feedstock.package_name)
+            )
```

Comment:
Maybe some notification should be sent out, as well? I'm open to ideas to improve it. Though blocking deployment due to some failed feedstock (particularly of orthogonal feedstocks), feels wrong to me.